apps/meteor/client/lib/rooms/roomTypes/livechat.ts

```diff
@@ -70,6 +70,9 @@ roomCoordinator.add(LivechatRoomType, {
 	},
 
 	findRoom(identifier) {
+		if (!hasPermission('view-l-room')) {
+			return null;
+		}
 		return ChatRoom.findOne({ _id: identifier });
```

Comment:
returning null here will trigger a new request to the server to load the room data.

Reply:
Done for all similar

Reply:
Undefined will have the same effect. If the room is not found locally it'll be loaded from the server.